netlogo-gui/src/main/app/infotab/InfoFormatter.scala

```diff
@@ -44,12 +44,12 @@ object InfoFormatter {
                          MaxParsingTimeMillis)
       .markdownToHtml(str)
 
-  def wrapHtml(body: HTML, fontSize: Int = defaultFontSize): HTML = {
-    "<html><head>"+styleSheet(fontSize)+"</head><body>"+body+"</body></html>"
-  }
+  def wrapHtml(body: HTML, fontSize: Int = defaultFontSize): HTML =
+    "<html><head>"+styleSheet(fontSize)+"</head><body>"+
+      body.replaceAll("<li>", "<li>&bull; ")+
```

Comment:
I'm not sure I understand. I believe the markdown parser generates `</li>`, but I didn't check. Why removing the closing tag would make the HTML valid, isn't the closing tag supposed to be there?

My only check was to edit the Info tab and see that a two items long list looks right, no HTML checking.